### DISCUSSION TITLE
no tenemos que ´programar nada en la primera kata de frontend?

### DISCUSSION THREAD

**Discussion by RupertoTacuche** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#62] [Q&A] [answered]
Hola a todos. Por lo que he podido ver, en esta primera kata no debemos programar nada, verdad?

**Comment by Hisahito** [comment, marked as answer]
Aun no. Estamos en la primera fase teórica/practica con cuestiones de planeación, requerimientos y diseño. Revisa bien la seccion de "Practicas", esa es la kata a entregar este próximo 25.

**Reply by RupertoTacuche** [reply]
Gracias amigo
